packages/story-editor/src/components/style/styleProvider.js

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useState, useRef, useEffect, useMemo } from '@googleforcreators/react';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../app/story';
+import { useHighlights } from '../../app/highlights';
+import { SELECTION, LINK, ANIMATION } from './constants';
+import Context from './context';
+import TextSelectionIcon from './textSelectionIcon';
+import ImageSelectionIcon from './imageSelectionIcon';
+import VideoSelectionIcon from './videoSelectionIcon';
+import ShapeSelectionIcon from './shapeSelectionIcon';
+import MultiSelectionIcon from './multiSelectionIcon';
```

Comment:
Maybe move these to a single file? Or at least a sub-folder? They're a bit noisy for such a tiny thing.